.github/workflows/test.yml

```diff
@@ -183,15 +184,15 @@ jobs:
         env:
           PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
-          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          SONATYPE_PASSWORD: ${{ secrets.CHIPSALLIANCE_SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.CHIPSALLIANCE_SONATYPE_USERNAME }}
 
```

Comment:
It turns out that chipsalliance has the `SONATYPE_USERNAME` and `SONATYPE_PASSWORD` set, but we override them with the settings for publishing under `edu.berkeley.cs`, and we can't really change that because old Chisel3 branches still need to be able to publish SNAPSHOTs.